community/cypher/cypher/src/test/scala/org/neo4j/cypher/ProfilerAcceptanceTest.scala

```diff
@@ -125,7 +125,7 @@ class ProfilerAcceptanceTest extends ExecutionEngineFunSuite with CreateTempFile
   test("tracks number of rows") {
     //GIVEN
     // due to the cost model, we need a bunch of nodes for the planner to pick a plan that does lookup by id
-    (1 to 10).foreach(_ => createNode())
+    (1 to 100).foreach(_ => createNode())
```

Comment:
So we actually need _more_ nodes to get the index to be picked now... Interesting.